apps/flame-defi/app/swap/page.tsx

```diff
@@ -408,11 +408,9 @@ export default function SwapPage(): React.ReactElement {
           />
         </ConfirmationModal>
         {(!userAccount.address || tokenApprovalNeeded) && (
-          <ActionButton
-            callback={onSubmitCallback}
-            buttonText={buttonText}
-            className="w-full mt-2"
-          />
+          <Button onClick={onSubmitCallback} className="w-full mt-2">
```

Comment:
This changes the styles. 
Existing buttons have white text and a gradient.
<img width="652" alt="Screenshot 2025-03-18 at 4 35 38 PM" src="https://github.com/user-attachments/assets/f0e72fe2-de80-47e9-ac1f-0ec6e187f074" />